go.mod

```diff
@@ -257,3 +257,5 @@ require (
 )
 
 replace github.com/tendermint/tendermint => github.com/bnb-chain/tendermint v0.31.12
+
+replace github.com/ledgerwatch/erigon-lib => github.com/hexoscott/erigon-lib v0.0.0-20221130104614-9b44654b2826
```

Comment:
likely need remove it

Reply:
granted you access to create PR's in erigon-lib repo

Reply:
There's a PR open already there for the changes to the txpool `Best` function

Reply:
The changes in erigon-lib are merged now so this is updated to the actual version from main now